api/spec/controllers/spree/api/line_items_controller_spec.rb

```diff
@@ -94,7 +94,9 @@ module Spree
         api_put :update, :id => line_item.id, :line_item => { :quantity => 101 }
         expect(response.status).to eq(200)
         order.reload
-        expect(order.total).to eq(1010) # 10 original due to factory, + 1000 in this test
+        subtotal = line_item.price * 101
+        shipping = order.shipments.first.cost
+        expect(order.total).to eq(subtotal + shipping)
```

Comment:
i generally find more comfort in the spec if we have an absolute value here rather than a calculated one